File: platform/sdk/python/examples/graph/graph-017_proj_community_in_1.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.projCommunityIn1("comm-001")
if not result["ok"] or result["nodes"] < 100:
    sys.exit(1)
print("XJ-OK: graph-017_proj_community_in_1")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-017_proj_community_in_1.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.projCommunityIn1("comm-001");
if (!result.ok || result.nodes < 100) process.exit(1);
console.log("XJ-OK: graph-017_proj_community_in_1");
process.exit(0);
